pages/common/accelerate.md

```diff
@@ -0,0 +1,28 @@
+# Accelerate
+
+> Accelerate is a library that enables the same PyTorch code to be run across any distributed configuration by adding just four lines of code.
```

Comment:
```suggestion
> Accelerate is a library that enables the same PyTorch code to be run across any distributed configuration.
```

For me, the `four lines` sounds like an advertisement.

Reply:
Yeah, it is intended to be an advertisement since I copied that directly from the description of the library.

I agree and will commit with your changes.